test/jquery.ime.test.fixtures.js

```diff
@@ -9,6 +9,30 @@ var palochkaVariants = {
 		digitOne: '1'
 	},
 	testFixtures = [ // eslint-disable-line no-unused-vars
+	{
+		description: 'Akan QX test',
+		inputmethod: 'ak-qx',
+		tests: [
+			{ input: 'Q', output: 'Ɛ', description: 'Akan QX Q' },
+			{ input: 'q', output: 'ɛ', description: 'Akan QX q' },
+			{ input: 'X', output: 'Ɔ', description: 'Akan QX X' },
+			{ input: 'x', output: 'ɔ', description: 'Akan QX x' },
+			{ input: '\\Q', output: 'Q', description: 'Akan QX \\Q' },
```

Comment:
Should this be \Q instead of \\Q and so on?

Reply:
JavaScript literal strings require '\\' when one backslash is needed.

Reply:
WAT. I'm pretty sure I wrote two. Trying again: '\\'.

Reply:
Funny GitHub, changes two backslashes to one.

Reply:
Anyway, I'm pretty sure that the code is correct.